pkg/tsdb/testdatasource/stream_handler.go

```diff
@@ -119,7 +119,7 @@ func (p *testStreamHandler) runTestStream(ctx context.Context, path string, conf
 
 			if flight != nil {
 				flight.set(0, conf.Flight.getNextPoint(t))
-				if err := sender.SendFrame(flight.frame, data.IncludeDataOnly); err != nil {
+				if err := sender.SendFrame(flight.frame, data.IncludeAll); err != nil {
```

Comment:
should we do `IncludeAll` vs `IncludeDataOnly` depending on the feature flag settings?

Reply:
Good catch, thx, I suppose yes - will do

Reply:
done